src/bindings/js/node/src/node_wrap.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
```suggestion
// Copyright (C) 2018-2025 Intel Corporation
// SPDX-License-Identifier: Apache-2.0

#pragma once
```
Add copyright to all new files, please.